chapters/packages.tex

```diff
@@ -227,12 +227,9 @@ \subsection{Directory Hierarchy Mapping}\label{mapping-a-package-class-hierarchy
 
 \subsection{Single File Mapping}\label{mapping-a-package-class-hierarchy-into-a-single-file-nonstructured-entity}\label{single-file-mapping}
 
-When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), that file shall only define a single class \lstinline!A! with a name matching the name of the nonstructured entity.
-In a file hierarchy the files shall have the extension \filename{.mo}.
-
-A \filename{.mo} file defining more than one class cannot be part of the mapping
-to file-structure and it is an error if it is loaded from the
-\lstinline!MODELICAPATH!.
+When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), the file content shall match \lstinline[language=grammar]!stored-definition! in the grammar.
+In this case, the \lstinline[language=grammar]!stored-definition! shall only define a single class with name (here, \lstinline!A!) matching the name of the nonstructured entity.
```

Comment:
So, what about:
```suggestion
In this case, the \lstinline[language=grammar]!stored-definition! shall only define a single class whose name (here, \lstinline!A!) matches the name of the nonstructured entity.
```

Reply:
Yes, that sounds better to me.  Applying.

Reply:
By the way, @eshmoylova, did this sound like an improvement of language to you?